### PR TITLE
I18n newdles participating

### PR DESCRIPTION
**Pull request by JHunterHobbs**
Added `<Trans>` `<Plural>` and `t` to text fields in NewdlesParticpating component to provide i18n capabilities. Tested `<Plural>` on Chrome browser in Spanish.

### CODE REVIEW

File: newdle/client/src/components/NewdlesParticipating.js [hunk 1]
```diff
@@ -24,7 +25,9 @@ export default function NewdlesParticipating() {
   } else if (participations.length === 0) {
     content = (
       <div className={styles['no-newdle-container']}>
-        <h2>You are not part of any newdles yet.</h2>
+        <Trans>
+          <h2>You are not part of any newdles yet.</h2>
```
Comment (ThiefMaster) [inline newdle/client/src/components/NewdlesParticipating.js:29]: Put the trans inside the h2 - no need to include the tag placeholders in the translatable string for cases where they are just surrounding the whole string

File: newdle/client/src/components/NewdlesParticipating.js [hunk 2]
```diff
@@ -98,11 +101,17 @@ function NewdleParticipation({
...
-              : `You replied with ${slotsChosen.length} timeslot(s)`}
+            {finalDT ? (
+              t`This newdle has finished`
+            ) : !slotsChosen.length ? (
+              t`Awaiting your reply`
+            ) : (
+              <Plural
+                value={slotsChosen.length}
+                one={`You replied with ${slotsChosen.length} timeslot`}
+                other={`You replied with ${slotsChosen.length} timeslots`}
```
Comment (ThiefMaster) [inline newdle/client/src/components/NewdlesParticipating.js:112]: you can use `#` as a placeholder for the `value` here

File: newdle/client/src/locales/es/messages.po [hunk 3]
```diff
@@ -13,6 +13,10 @@ msgstr ""
 "Language-Team: \n"
 "Plural-Forms: \n"
 
+#: src/components/NewdlesParticipating.js:28
+msgid "<0>You are not part of any newdles yet.</0>"
```
Comment (ThiefMaster) [inline newdle/client/src/locales/es/messages.po:17]: just fyi, the `<0>...</0>` around this will disappear once you fix the nesting with h2/trans

**Review by ThiefMaster — CHANGES_REQUESTED** [review]

File: newdle/client/src/components/NewdlesParticipating.js (hunk 1 above)
Comment (JHunterHobbs) [reply newdle/client/src/components/NewdlesParticipating.js]: Oh yea. oops... Fixed. Thanks!

File: newdle/client/src/components/NewdlesParticipating.js (hunk 2 above)
Comment (JHunterHobbs) [reply newdle/client/src/components/NewdlesParticipating.js]: Addressed

**Comment by ThiefMaster** [comment]
Thanks! Would you mind rebasing to the latest master? We added an eslint rule for consistent import sorting, so there are some conflicts now :/

**Comment by JHunterHobbs** [comment]
Whew..., ok think I got it rebased correctly. Please let me know if something is off.

**Comment by ThiefMaster** [comment]
Thanks! As a tip to save some sanity the next time: First squash all your commits into a single one and then rebase that commit. Only one commit with conflicts to resolve that way, and in most cases the individual commits of small PRs are not useful anyway.

**Pull request merged**